tests/basic/storage_pool_test.go

```diff
@@ -9762,3 +9762,106 @@ var _ = Describe("{AddDriveMetadataPool}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+var _ = Describe("{PoolExpandRebalanceShutdownNode}", func() {
+	/*
+		1. create one pool
+		2. run ios
+		3. expand pool to higher size
+		4. while it is in progress shutdown and restart the node
+		5. let rebalance continue.
+		6. check if pending operation continues
+	*/
+
+	var testrailID = 0
+	// Testrail Description : while pool expand Rebalance is in progress ShutdownNode and check operation resumes
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("PoolExpandRebalanceShutdownNode",
+			"Get Px State after pool delete",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+	stepLog := "Get Px State after pool delete"
+	It(stepLog, func() {
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("rebalanceshutdown-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get Pool with running IO on the cluster
+		poolUUID, err := GetPoolIDWithIOs(contexts)
+		log.FailOnError(err, "Failed to get pool running with IO")
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+
+		// Get Node Details of the Pool with IO
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+		log.InfoD("Pool with UUID [%v] present in Node [%v]", poolUUID, nodeDetail.Name)
+
+		if IsLocalCluster(*nodeDetail) != true {
```

Comment:
can you please move this step before scheduling  context ?

Reply:
this cannot be moved before scheduling context as nodedetail is required and is taken before this step.So moving this as done for now